src/firebase/actionInDatabase/addedUserToDatabase.ts

```diff
@@ -0,0 +1,13 @@
+import { doc, setDoc } from 'firebase/firestore'
+import { firebaseCollection, User } from '../../types'
+import { db } from '../firebase'
+
+export const addedUserToDatabase = (user: User) => {
+  const { userName, userUUID, roomUUID } = user
+
+  setDoc(doc(db, firebaseCollection.users, userUUID), {
```

Comment:
imo, you should return it to handle somehow error. Promise without a catch is a serious mistake